app/views/admin/accounts/_form.html.erb

```diff
@@ -18,6 +18,13 @@
     <% end %>
 
     <%= f.inputs do %>
+      <%= theming.input :default_locale,
+                        as: :select,
+                        include_blank: false,
+                        collection: Pageflow.config.available_public_locales.map{ |locale|
+                          [I18n.t("pageflow.languages.#{locale}"), locale.to_s]
```

Comment:
```suggestion
                          [I18n.t('pageflow.public._language', locale: locale), locale.to_s]
```

see https://github.com/codevise/pageflow/blob/f92c4352a375fcbfba50739aac6d1e52ddb496c3/app/assets/javascripts/pageflow/editor/views/edit_meta_data_view.js#L29-L34

Reply:
I saw that, but that would give us the local name of the locale. Since the user of the admin panel is either german or english, I thought it would be better to translate the names of the locales to the language specified for the current user

Reply:
On the other hand that'S also how it works when selecting a locale inside the editor interface (which is also only either german or english). Also this would mean we would have to release a new version of the Pageflow gem whenever a locale is added to the `pageflow-public-i18n`, which would defeat one of the main reasons to decouple translations into their own gem.

Reply:
True. Ok, I'll change it to use the local names like in the editor